src/helpers/dom/virtualAttributes.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @class Attr
+ */
+class Attr {
+  constructor(name, value) {
+    this.name = name;
+    this.value = value;
+  }
+}
+
+/**
+ * @class VirtualAttributes
+ */
+class VirtualAttributes {
+  /**
+   * Constructor
+   *
+   * @param {VirtualElement} element - instance.
+   * @param {Object} [attributes] - default set of attributes.
+   */
+  constructor(element, attributes) {
+    this._element = element;
+    this.length = 0;
+
+    if (typeof attributes === 'object') {
+      const keys = Object.keys(attributes);
+      const len = keys.length;
+
+      for (let i = 0, n = 0, name; i < len; i++) {
+        name = keys[i];
+        if (typeof this[name] !== 'function') {
+          this[n++] = this[name] = new Attr(name, attributes[name]);
```

Comment:
I think it is better to create a new property (array) instead of using the instance as it was an array.